server/service/client.go

```diff
@@ -2008,6 +2008,11 @@ func (c *Client) DoGitOps(
 		} else {
 			mdmAppConfig["enable_disk_encryption"] = false
 		}
+		if config.Controls.RequireBitLockerPIN != nil {
```

Comment:
Do we want to do any kind of validation here to make sure that if `enable_disk_encryption` is false then `windows_require_bitlocker_pin` can't be true?